# William Barker, PhD

## About Me
I am a PhD Chemist and Microbiologist with a strong passion for leveraging data science and machine learning in the pharmaceutical industry. My academic and professional journey has equipped me with a deep understanding of these domains, and I am skilled in Python, SQL, R, and Bash scripting. I specialize in applying machine learning techniques to complex biological and chemical datasets to gain actionable insights. I enjoy using data as a storytelling tool.

## Skills
- **Programming Languages:** Python, R, SQL, Bash
- **Machine Learning Libraries:** TensorFlow, PyTorch, scikit-learn, Pandas, NumPy, SciPy, Matplotlib, seaborn
- **Machine Learning Modalities:** LLMs, LSTM, Transformers, Graphs, Convolutional Neural Networks
- **Big Data Management:** MySQL, PostgreSQL, Spark, Airflow, Hadoop, AWS (S3, Lambda, Sagemaker), GCP (BigQuery)

## Experience
### Scientist: CSL Seqirus - 2021-Present
- Spearheading Machine learning and artificial intelligence implementations that provide insights into the stochastic nature of viruses and virus-host interactions leading to increased vaccine yields in manufacturing.
- Pioneered research studying host cell and influenza virus interactions using next-generation sequencing and advanced R programming (dpylr, ggplot2, DESeq2) for insightful data analysis.
- Innovated vaccine yield prediction using ProteinBERT, a state-of-the-art large language model for protein sequences, fine-tuned on known influenza protein sequences.
- Developed a comprehensive pipeline to analyze and visualize influenza mutation patterns across the entire genome, enhancing understanding of yield fluctuations.
- Improved seasonal influenza vaccine dose yields (Flucelvax) by creating transgenic viruses with enhanced growth kinetics in an MDCK cell-based manufacturing system.
- Leveraged reverse genetics to generate viral seed stocks under GMP controls (BSL2+/BSL3) in support of a $60MM annual government contract to manufacture pilot batches of vaccine for novel and endemic influenza.
- Initiated a multi-divisional project aimed at improving mammalian cell line performance by screening small molecule additives that restrict the innate antiviral immune response.
- Supported departmental needs by volunteering on committees with oversight on the construction of new lab space, as well as company safety. Provided subject matter expertise on the safe handling and storage of chemical reagents.

### Lead Scientist: Agile Sciences - 2020-2021
- Designed and performed biological assays that utilized small molecules to defeat antibiotic resistance and defined novel strategies for treating problematic bacterial species in human and animal health: ESKAPE pathogens, Salmonella, Mycobacterium, Neisseria, Campylobacter, and many other fastidious bacterial species.
- Leveraged an existing collection of over 500 proprietary molecules to create a chemical library formatted for high throughput screening methods; implemented this library to find numerous hit molecules with significant potency against multi-drug resistant bacteria.
- Expanded the known activity spectrum of the company lead molecule in pre-IND stage through comprehensive screening.
- Supported IND and NADA-enabling studies by providing subject matter expertise on company core intellectual property and bacterial physiology.

### PhD Candidate, Univ of Notre Dame - 2016-2020
- Synthesized and optimized small molecules that reverse antibiotic resistance and modify bacterial behavior in clinically relevant pathogens.
- Collaborated with 5 research groups and organized interdisciplinary efforts leading to 8 publications. Directly established two collaborations to enhance research quality.
- Revised in-house laboratory safety protocols and implemented engineering controls for hazardous waste to maintain full compliance with health and safety regulations.

### Research Assistant, UNC-Chapel Hill - 2010-2016
- Designed and constructed gene targeting vectors using recombinant DNA technology in embryonic stem cells to create mouse models of human genetic diseases.
- Validated gene sequence and function of targeted loci through standard PCR, Southern/Northern Blots, and RT-PCR techniques.

## Education
- PhD Chemistry, Notre Dame, 2020
- BS Chemistry, University of North Carolina at Chapel Hill, 2013

## Personal Projects
- [PorscheChat](https://github.com/williamtbarker/porschechat) - An LLM chatbot pre-trained on data related to Porsche automobiles. (Django, Flask, React)
- [Porsche 911 Auction EDA](https://www.kaggle.com/williamtbarker/porsche-911-auction-eda/) - Exploratory Data Analysis and Plots of scraped car auction data. (Jupyter)

## Publications
- J Bailey, LA Gallagher, WT Barker, VB Hubble, J Gasper, C Melander, and C Manoil. “Genetic Dissection of Antibiotic Adjuvant Activity.” mBio, January 2022.
- WT Barker. “Small Molecules that Synergize with Colistin” Doctoral Dissertation, University of Notre Dame. June 2020.
- WT Barker, LA Jania, RJ Melander, BH Koller, and C Melander. “Eukaryotic Phosphatase Inhibitors Enhance Colistin Efficacy in Gram-negative Bacteria.” Chemical Biology and Drug Design. June 2020.
- AM Nemeth, AK Basak, AW Weig, SA Marrujo, WT Barker, LA Jania, TA Hendricks, AE Sullivan, PM O’Connor, RJ Melander, BH Koller, and C Melander. “Structure-Function Studies on IMD-0354 Identifies Highly Active Colistin Adjuvants.” ChemMedChem, November 2019.
- WT Barker, AM Nemeth, SM Brackett, AK Basak, CE Chandler, WJ Zuercher, RJ Melander, RK Ernst, and C Melander. “Repurposing Eukaryotic Kinase Inhibitors as Colistin Adjuvants in Gram-negative Bacteria.” ACS Infectious Diseases, August 2019.
- BM Minrovic, VB Hubble, WT Barker, LA Jania, RJ Melander, BH Koller, and C Melander. “Second-Generation Tryptamine Derivatives Potently Sensitize Colistin-resistant Bacteria to Colistin.” ACS Medicinal Chemistry Letters, April 2019.
- WT Barker, CE Chandler, RJ Melander, RK Ernst, and C Melander. “Tryptamine Derivatives Disarm Colistin Resistance in Polymyxin-resistant Gram-negative Bacteria.” Bioorganic and Medicinal Chemistry, March 2019.
- WM Huggins, WT Barker, JT Baker, NA Hanh, RJ Melander, and C Melander. “Meridianin D Analogs Display Antibiofilm Activity Against MRSA and Increase Colistin Efficacy in Gram-Negative Bacteria.” ACS Medicinal Chemistry Letters, May 2018.
- WT Barker, SE Martin, CE Chandler, TV Nguyen, TL Harris, C Goodell, RJ Melander, Y Doi, RK Ernst, and C Melander. “Small Molecule Adjuvants that Suppress both Chromosomal and mcr-1 Encoded Colistin Resistance and Amplify Colistin Efficacy in Polymyxin-susceptible Bacteria.” Bioorganic and Medicinal Chemistry, October 2017.


## Contact Me
- Email: barkerwilliamt@gmail.com
- LinkedIn: [William T. Barker](https://www.linkedin.com/in/williamtbarker)
- GitHub: [@williamtbarker](https://williamtbarker.github.io)
- Kaggle: [William T. Barker](https://www.kaggle.com/williamtbarker)


This README serves as a snapshot of my expertise in the intersection of chemistry, virology, and machine learning. I am always interested in collaborative opportunities that leverage these skills to address challenging problems in science and healthcare.
